pom.xml

```diff
@@ -76,5 +76,17 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>cloud.orbit</groupId>
+            <artifactId>orbit-spring</artifactId>
```

Comment:
This project shouldn't have a dependency on Spring. What we should have is within orbit-spring the ability to detect that the dropwizard metrics library is on the classpath and autoconfigure things from there.

https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-developing-auto-configuration.html